devtools/create_project/create_project.cpp

```diff
@@ -1297,6 +1298,16 @@ static void fixupFeatures(ProjectType projectType, BuildSetup &setup) {
 		setFeatureBuildState("mikmod", setup.features, false);
 	}
 
+	// With libcurl we get an HTTP client
+	if (!getFeatureBuildState("libcurl", setup.features)) {
+		setFeatureBuildState("http", setup.features, false);
+	}
```

Comment:
should this be
```
if (getFeatureBuildState("libcurl", setup.features)) {
	setFeatureBuildState("http", setup.features, true);
}
```
according to comment?

should there be another `if` that checks for emscripten and fixes value of http too?

Reply:
http feature is on by default so not having libcurl makes it disabled.
I will change the comment.

For emscripten, no need, as the platform doesn't use create_project I suppose.